docs/ja/structure.md

```diff
@@ -0,0 +1,84 @@
+# バリデーション結果構造
+
+バリデーション結果はこの構造でアクセスできます:
+
+```
+{
+  // top-level validation properties
+  valid: true,
+  invalid: false,
+  touched: false,
+  undefined: true,
+  dirty: false,
+  pristine: true,
+  modified: false,
+  errors: [{
+    field: 'field1', validator: 'required', message: 'required field1'
+  }, ... {
+    field: 'fieldX', validator: 'customValidator', message: 'invalid fieldX'
+  }],
+
+  // field1 validation
+  field1: {
+    required: false, // build-in validator, return `false` or `true`
+    email: true, // custom validator
+    url: 'invalid url format', // custom validator, if specify the error message in validation rule, set it
+    ...
+    customValidator1: false, // custom validator
+    // field validation properties
+    valid: false,
+    invalid: true,
+    touched: false,
+    undefined: true,
+    dirty: false,
+    pristine: true,
+    modified: false,
+    errors: [{
+      validator: 'required', message: 'required field1'
+    }]
+  },
+
+  ...
+
+  // fieldX validation
+  fieldX: {
+    min: false, // validator
+    ...
+    customValidator: true,
+
+    // fieldX validation properties
+    valid: false,
+    invalid: true,
+    touched: true,
+    undefined: false,
+    dirty: true,
+    pristine: false,
+    modified: true,
+    errors: [{
+      validator: 'customValidator', message: 'invalid fieldX'
+    }]
+  },
+}
+```
+
+様々なトップレベルプロパティはバリデーションスコープの中に、そして各フィールドのバリデーション結果は自身それぞれのスコープの中にあります。
+
+## フィールドバリデーションプロパティ
+- `valid`: フィールドが有効化どうか; もし有効なら、`true` を返し、そうでなければ `false` を返します。
```

Comment:
I see the errors by textlink for `;`. 
```sh
$ textlint structure.md

/Users/hypermkt/work/vue-validator/docs/ja/structure.md
  67:24  error  原則として和文ではセミコロン(;)を使用しません。  preset-jtf-style/4.2.8.セミコロン(;)
  71:31  error  原則として和文ではセミコロン(;)を使用しません。  preset-jtf-style/4.2.8.セミコロン(;)
  72:36  error  原則として和文ではセミコロン(;)を使用しません。  preset-jtf-style/4.2.8.セミコロン(;)
  77:31  error  原則として和文ではセミコロン(;)を使用しません。  preset-jtf-style/4.2.8.セミコロン(;)
```

> ただし、文の途中にハイフン- や セミコン; がある場合は、その記号があると理解しづらい訳になる場合は、例外として削除てもよいです。

Maybe I should delete `;` this time. It doesn't look natural.